tests/ml_models/test_sentencetransformermodel_pytest.py

```diff
@@ -452,5 +453,37 @@ def test_missing_fields_in_config_json():
     clean_test_folder(TEST_FOLDER)
 
 
+def test_truncation_parameter():
+    model_id = "sentence-transformers/msmarco-distilbert-base-tas-b"
+
+    clean_test_folder(TEST_FOLDER)
+    test_model10 = SentenceTransformerModel(
+        folder_path=TEST_FOLDER,
+        model_id=model_id,
+    )
+
+    test_model10.save_as_pt(model_id=model_id, sentences=["today is sunny"])
+
+    tokenizer_json_file_path = os.path.join(TEST_FOLDER, "tokenizer.json")
+    try:
+        with open(tokenizer_json_file_path, "r") as json_file:
+            tokenizer_json = json.load(json_file)
+    except Exception as exec:
+        assert (
+            False
+        ), f"Creating tokenizer.json file for tracing raised an exception {exec}"
+
+    assert tokenizer_json[
+        "truncation"
+    ], "truncation parameter in tokenizer.json is null"
+
+    model11 = SentenceTransformer(model_id)
```

Comment:
Oh yeah, this is `SentenceTransformerModel` class not `SentenceTransformer`. In that case let's match with static value as we know the value already. I don't want to load models unnecessary as this can increase the execution time of integration tests.

Reply:
Got it. Should I remove the last commit and make one more commit or can I make it without removing?

Reply:
You don't need to remove the last commit, you can push another commit with the modification.

Reply:
Name the variable: `MAX_LENGTH_TASB` and then assert that. please don't just assert with a number.

Reply:
Done